src/iracingdataapi/client.py

```diff
@@ -1204,10 +1305,25 @@ def get_series_assets(self):
 
     def series_assets(self):
         print(
-            "series_assets() is deprecated and will be removed in a future release, please update to use get_series_assets"
+            "series_assets() is deprecated and will be removed in a future release, "
+            "please update to use get_series_assets"
         )
         return self.get_series_assets()
 
+    def series_past_seasons(self, series_id):
+        """Get all seasons for a series.
+
+        Filter list by ``'official'``: ``True`` for seasons with standings.
+
+        Args:
+            series_id ():
+
+        Returns:
+            dict: a dict containing information about the series and a list of seasons.
+        """
+        payload = {"series_id": series_id}
+        return self._get_resource("/data/series/past_seasons", payload=payload)
```

Comment:
I think it could be good to only return the `series` part of the response, having checked for `success` beforehand. Similar to how you did in `member_awards`
```suggestion
        resource = self._get_resource("/data/series/past_seasons", payload=payload)
        
        if resource.get('data', {}).get('success'):
            return resource['series']
        else:
            return {}
```

Also probably better to return something more insightful when there's an unsuccessful call.

Reply:
Funny, I was just running into this while trying to implement this in my bot.  Good idea.  I'll push an update later today when I can get to it.

As far as the return value in an unsuccessful call... I'm not really sure how the /data API works.  With `member_awards`, if I try a random number as garbage input I get an empty list in return and a value of 'success', so my check for `success` is probably not really needed.  With `series_past_seasons`, if I try garbage input I get a RuntimeError for an Unhandled Non-200 response (Response [404]) and I'm not really sure how to handle that.

I could probably just pull the check from both methods and leave error handling to the user to ensure the input is valid.